src/config.ts

```diff
@@ -1,3 +1,31 @@
+import log from "npmlog";
+
 export function getPort(): number {
   return process.env.PORT ? parseInt(process.env.PORT, 10) : 3000;
 }
+
+export function getProxyHostPort(): string {
+  const proxyHostPort = process.env.PROXY_HOST_PORT;
```

Comment:
can we please think about some kind of CLI interface ASAP (meow?). I _really_ dont like this approach. :(